### PR TITLE
feat(get): Add 'get rds' command

### LINKED ISSUE #17: [FEATURE] Add `awsctl get rds` command
### Description

Command to get information about rds

```
awsctl get rds
```

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by amalmadhu06**
Added get rds command which retrieves useful information related to RDS instances such as Instance ID, Engine, Status and Endpoint. 

## Fixes Issue
Closes #17 

## Changes proposed

Added get rds command

## Screenshots

image

## Note to reviewers
I have added Instance ID, Engine, Status, and Endpoint in the output assuming that they are useful information.

### CODE REVIEW

File: pkg/getRdsInfo.go
Comment (surajnarwade) [inline pkg/getRdsInfo.go:1]: can this go inside `/pkg/rds/rds.go`

Comment (amalmadhu06) [reply pkg/getRdsInfo.go:1]: Sure. On it

**Comment by amalmadhu06** [comment]
- Removed pkg.getRdsInfo.go,  moved code to pkg/rds.rds.go.
- Added  VERSION in output

image

File: cmd/rds.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	}
+
+	if len(rdsInstances) == 0 {
+		fmt.Println("No RDS instances found")
+		return nil
+	}
+	w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+	defer w.Flush()
+
+	fmt.Fprintln(w, "DB_INSTANCE_ID\tSTATUS\tENDPOINT\tENGINE\tVERSION")
```
Comment (surajnarwade) [inline cmd/rds.go:41]: this will be slightly different, check other commands

Comment (amalmadhu06) [reply cmd/rds.go]: changed
 `fmt.Fprintln(w, "DB_INSTANCE_ID\tSTATUS\tENDPOINT\tENGINE\tVERSION")` 
to 
`fmt.Fprintln(w, "DB_INSTANCE_ID", "\t", "STATUS", "\t", "ENDPOINT", "\t", "ENGINE", "\t", "VERSION")`

File: cmd/rds.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		fmt.Println("No RDS instances found")
+		return nil
+	}
+	w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+	defer w.Flush()
+
+	fmt.Fprintln(w, "DB_INSTANCE_ID", "\t", "STATUS", "\t", "ENDPOINT", "\t", "ENGINE", "\t", "VERSION")
+
+	for _, i := range rdsInstances {
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", i.DBInstanceID, i.Status, i.Endpoint, i.Engine, i.Version)
```
Comment (surajnarwade) [inline cmd/rds.go:44]: can you do the same for this as well, it makes code easier to read.

Comment (amalmadhu06) [reply cmd/rds.go]: Removed  `fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", i.DBInstanceID, i.Status, i.Endpoint, i.Engine, i.Version)` and included 
`fmt.Fprintln(
			w,
			i.DBInstanceID, "\t",
			i.Status, "\t",
			i.Endpoint, "\t",
			i.Engine, "\t",
			i.Version, "\t",
		)`

**Review by surajnarwade — APPROVED** [review]
Thank you for your contribution @amalmadhu06 this LGTM 🎉

**Pull request merged**